<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapCreatorHeightMapDialog</class>
 <widget class="QWidget" name="MapCreatorHeightMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HeightMap</string>
  </property>
  <widget class="QLabel" name="labelPersistence">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>213</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Persistence:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOctaveCount">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFrequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>173</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelOctaveCount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>133</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Octave count:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXBoundLower">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelBounds">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>53</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Bounds:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPersistence">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFrequency">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCreate">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>120</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMapSize">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Map size:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdiMapSizeX">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdiMapSizeY">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditXBoundUpper">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditZBoundLower">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditZBoundUpper">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdiMapSizeX</tabstop>
  <tabstop>lineEdiMapSizeY</tabstop>
  <tabstop>lineEditXBoundLower</tabstop>
  <tabstop>lineEditXBoundUpper</tabstop>
  <tabstop>lineEditZBoundLower</tabstop>
  <tabstop>lineEditZBoundUpper</tabstop>
  <tabstop>lineEditOctaveCount</tabstop>
  <tabstop>lineEditFrequency</tabstop>
  <tabstop>lineEditPersistence</tabstop>
  <tabstop>pushButtonCreate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
